### PR TITLE
Add header playground

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes BLUI-5889.

#### Changes proposed in this Pull Request:

- add header playground
-
-

#### Screenshots / Screen Recording (if applicable)

image

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: docs/src/componentDocs/Header/playground/PlaygroundPage.tsx [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+                maxHeight: '100%',
+                maxWidth: '100%',
+                position: 'relative',
+            }}
+            ref={containerRef}
+        >
+            <Header
+                style={{ width: DRAWER_WIDTH, margin: 'auto' }}
+                {...rest}
+                title={'Valley Forge'}
```
Comment (ArshdeepSingh3-eaton) [inline docs/src/componentDocs/Header/playground/PlaygroundPage.tsx:209]: remove title and subtitle

File: docs/src/componentDocs/Header/playground/PlaygroundPage.tsx [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+                subtitle={'The Last Stand'}
+                icon={getIcon(icon as unknown as string)}
+                actionItems={getActionItems(actionItems as unknown as boolean)}
+                // searchableConfig={{ onChangeText: () => {} }}
+                backgroundImage={getImage(backgroundImage?.toString() ?? '')}
+                variant="static"
+            />
```
Comment (ArshdeepSingh3-eaton) [inline docs/src/componentDocs/Header/playground/PlaygroundPage.tsx:216]: add the following styles `styles={{backgroundImage:{width: DRAWER_WIDTH, margin: 'auto' }}}`

**Review by ArshdeepSingh3-eaton — COMMENTED** [review]
everything else looks good

**Comment by KomalGDhere-eaton** [comment]
Looks good to me.

**Review by KomalGDhere-eaton — APPROVED** [review]

**Pull request merged**